addons/mail/static/tests/owl/component/thread_tests.js

```diff
@@ -82,7 +96,222 @@ QUnit.test('dragover files on thread with composer', async function (assert) {
     assert.ok(
         document.querySelector('.o_Composer_dropZone'),
         "should have dropzone when dragging file over the thread");
+});
+
+QUnit.only('message list desc order', async function (assert) {
+    assert.expect(5);
+    let amountOfCalls = 0;
+    let lastId = 10000;
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'message_fetch') {
+                if (amountOfCalls > 4) {
+                    return [];
+                }
+                // multiple calls here to be able to test load more (up to (10000 / 30) calls normally
+                let messagesData = [];
+                const amountOfMessages = 30;
+                const firstIValue = (lastId - amountOfCalls * amountOfMessages) - 1;
+                const lastIValue = firstIValue - amountOfMessages;
+
+                for (let i = firstIValue; i > lastIValue; i--) {
+                    messagesData.push({
+                        author_id: [firstIValue, `#${firstIValue}`],
+                        body: `<em>Page ${amountOfCalls + 1}</em><br/><p>#${i} message</p>`,
+                        channel_ids: [20],
+                        date: "2019-04-20 10:00:00",
+                        id: lastId + i,
+                        message_type: 'comment',
+                        model: 'mail.channel',
+                        record_name: 'General',
+                        res_id: 20,
+                    });
+                }
+                lastId = lastIValue;
+                amountOfCalls++;
+                return messagesData;
+            }
+            return this._super(...arguments);
+        }
+    });
+    const threadLocalId = this.store.dispatch('_createThread', {
+        channel_type: 'channel',
+        id: 100,
+        members: [
+            {
+                email: "john@example.com",
+                id: 9,
+                name: "John",
+            },
+            {
+                email: "fred@example.com",
+                id: 10,
+                name: "Fred",
+            }
+        ],
+        name: "General",
+        public: 'public',
+    });
+    await this.createThread(threadLocalId, {
+        order: 'desc'
+    });
+    // needed to allow scrolling
+    this.thread.el.style.height = '300px';
+    await testUtils.nextTick();
+    assert.strictEqual(
+        document
+            .querySelectorAll(`.o_MessageList_loadMore + .o_Message`)
```

Comment:
only problem with this selector is that it forces messages and loadmore being siblings, which is the case now but this is an implementation detail